modules/tide_site_preview/src/Plugin/Block/PreviewLinksBlock.php

```diff
@@ -234,8 +234,7 @@ protected function getCurrentNode() : ?NodeInterface {
     if ($route_name === 'entity.node.revision') {
       try {
         $vid = $this->routeMatch->getParameter('node_revision');
```

Comment:
This will directly return the node revision entity post Drupal 9.3.x

Reply:
yes, I think that is why it used to work, but after updating to 9.4.x it started returning object as well. I have seen similar issues raised against d10 as well. Hopefully that will at least fix the issue from our side. @govindmaloo